python/paddle/distributed/auto_parallel/api.py

```diff
@@ -1425,3 +1425,65 @@ def to_static(
     dist_loader = dist_model.dist_loader
 
     return dist_model, dist_loader
+
+
+def unshard_dtensor(dist_tensor):
+    """
+    Converts a distributed tensor to its original dense tensor. ``unshard_dtensor``
```

Comment:
It seems `original dense tensor` is a little bit confusing. Actually, it does 2 things,
1. Make it Replicated on all mesh dims
2. Convert to Paddle.Tensor

Reply:
Done, modified the doc.